fluent/src/index.js

```diff
@@ -7,9 +7,8 @@
  *
  */
 
-export { default as _parse } from "./parser";
-
-export { FluentBundle } from "./context";
+export { default as FluentBundle } from "./context";
```

Comment:
I'll rename `context.js` to `bundle.js` in a follow-up commit (without a separate PR).